src/OrbitGl/App.cpp

```diff
@@ -2042,6 +2043,19 @@ void OrbitApp::LoadPreset(const PresetFile& preset_file) {
   });
 }
 
+void OrbitApp::ShowPresetInExplorer(const PresetFile& preset) {
+  const auto file_exists = orbit_base::FileExists(preset.file_path());
+  if (file_exists.has_error()) {
+    ERROR("Unable to find preset file: %s", file_exists.error().message());
+    return;
+  }
+
+  std::string command = absl::StrFormat("explorer.exe /select,\"%s\"", preset.file_path().string());
```

Comment:
Does this work on Linux?

Reply:
It only works for windows and this is also one of the reason that I add it to `OrbitApp` rather than the file utils in `OrbitBase` which supports both windows and linux.

Also we don't open Orbit client on Linux. So this might be fine XD

Reply:
Half the team use Orbit client on Linux. Am I missing something? @beckerhe

Reply:
Nevertheless we should not deliberately break the Linux UI. We could just make a Windows-Linux-distinction and call a different command on Linux. This one should work:

```
dbus-send --session --print-reply --dest=org.freedesktop.FileManager1 --type=method_call /org/freedesktop/FileManager1 org.freedesktop.FileManager1.ShowItems array:string:"file:///%s" string:""
```

Reply:
Thanks for the suggestion. If so I will move this method to `OrbitBase/File.h` and make it support both windows and linux

Reply:
I believe we don't have an easy way to start a process in OrbitBase (except using `system` or other low-level system APIs which is not ideal / a lot of work / hard to get right)

Reply:
I'm not opposed to that but you have to willing to spend some time to understand the Posix and Windows APIs to make it work properly. :slightly_smiling_face:

Reply:
😂 make sense. I will keep it here